doc/source/examples/cfd-dem/square-fluidized-bed/square-fluidized-bed.rst

```diff
@@ -0,0 +1,389 @@
+        set output path                     = ./output/
+end
+
+
+
+Physical properties
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The physical properties subsection allows us to determine the density and viscosity of the fluid. We choose a density of 1 and viscosity of 0.00001 as to simulate the flow of air. 
+
+.. code-block:: text
+
+    subsection physical properties
+        subsection fluid 0
+            set kinematic viscosity            = 0.00001
+            set density                        = 1
+          end
+    end
+
+
+Initial conditions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For the initial conditions, we choose zero initial conditions for the velocity. 
+
+.. code-block:: text
+
+    subsection initial conditions
+        set type = nodal
+        subsection uvwp
+            set Function expression = 0; 0; 0; 0
+        end
+    end
+ 
+
+Boundary conditions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For the boundary conditions, we choose a slip boundary condition on the walls of the cylinder (ID = 0) and an inlet velocity of 0.2 m/s at the lower face of the bed (ID = 2).
+
+.. code-block:: text
+
+    subsection boundary conditions
+       set number                  = 5
+   	subsection bc 0
+     	    set id		   = 0
+     	    set type              = slip
+        end
+       subsection bc 1
+           set id		   = 1
+           set type              = slip
+        end
+       subsection bc 2
+           set id		   = 4
+           set type              = slip
+        end
+       subsection bc 3
+           set id		   = 5
+           set type              = slip
+        end
+       subsection bc 4
+           set id		   = 2
+           set type              = function
+           subsection u
+            set Function expression = 0
+           end
+           subsection v
+            set Function expression = 2
+           end
+           subsection w
+            set Function expression = 0
+           end
+       end
+    end
+
+The additional sections for the CFD-DEM simulations are the void fraction subsection and the CFD-DEM subsection. These subsections are described in detail in the `CFD-DEM parameters <../../../parameters/unresolved_cfd-dem/unresolved_cfd-dem.html>`_ .
+
+Void fraction
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Since we are calculating the void fraction using the packed bed of the DEM simulation, we set the mode to "dem". For this, we need to read the dem files which we already wrote using check-pointing. We therefore set the read dem to "true" and specify the prefix of the dem files to be read. In order to ensure that our void fraction projection is bounded, we choose an upper bound limit of 1. We decide not to lower bound the void fraction and thus attributed a value of 0 to the L2 lower bound parameter. We now choose a smoothing factor for the void fraction as to reduce discontinuity which can lead to oscillations in the velocity. The factor we choose is around the square of twice the particle's diameter. 
```

Comment:
```suggestion
Since we are calculating the void fraction using the packed bed of the DEM simulation, we set the mode to "dem". For this, we need to read the dem files which we already wrote using check-pointing. We therefore set the read dem to "true" and specify the prefix of the dem files to be dem. In order to ensure that our void fraction projection is bounded, we choose an upper bound limit of 1. We decide not to lower bound the void fraction and thus attributed a value of 0 to the L2 lower bound parameter. We now choose a smoothing factor for the void fraction as to reduce discontinuity which can lead to oscillations in the velocity. The factor we choose is around the square of twice the particle's diameter. 
```